### ISSUE TITLE
appstream error from Debian

### ISSUE THREAD

**Issue by yuzibo** [neomutt/neomutt#3278]
Hi,
I am not sure  this is the right place to report it :-) 
But i  found the appstream error accidentially from Debian about neomutt form [here](https://appstream.debian.org/sid/main/issues/neomutt.html)
And I want to fix it, but can not find the Debian branch.

**Comment by dcpurton** [comment]
Would this be a packaging issue?

You can submit Debian bug reports using the [`reportbug`](https://wiki.debian.org/reportbug) tool.

**Comment by flatcap** [comment]
Hmm... I've wondered about installing a `.desktop` file.

We don't at the moment, which means Debian must be creating one.
We _do_ have icons, though, in our [contrib/logo](https://github.com/neomutt/neomutt/tree/master/contrib/logo) dir.
They get installed to `/usr/share/doc/neomutt/logo/`

If someone can find the correct place for them, we can fix the install.

Can you post the `.desktop` file that Debian is using.
Also, can you find out if there are any installation commands that a `.desktop` file needs.

If we can agree on the contents, we could update our install.

My experiments are here: https://gist.github.com/flatcap/a1c94399c40893db39d641fb106a0dc2

Thanks

**Comment by dcpurton** [comment]
The current Debian `.desktop` file is:

```
[Desktop Entry]
Categories=Office;Network;Email;
Comment=Simple text-based Mail User Agent
Comment[de]=Einfaches, Text-basiertes Mailprogramm
Exec=neomutt %u
Icon=neomutt
Name=neomutt
Name[de]=Mutt
MimeType=x-scheme-handler/mailto;
NoDisplay=false
Terminal=true
Type=Application
```

**Comment by flatcap** [comment]
Oh, translations!
I hadn't thought about that.

**Comment by dcpurton** [comment]
The icons are installed in `/usr/share/doc/neomutt/logo/` which is not terribly helpful. There's nothing in `/usr/share/icons/` or any of its subdirectories.

**Comment by dcpurton** [comment]
## Icon placement

`contrib/logo/neomutt-16.png` → `/usr/share/icons/hicolor/16x16/apps/neomutt.png`
`contrib/logo/neomutt-32.png` → `/usr/share/icons/hicolor/32x32/apps/neomutt.png`
`contrib/logo/neomutt-64.png` → `/usr/share/icons/hicolor/64x64/apps/neomutt.png`
`contrib/logo/neomutt-128.png` → `/usr/share/icons/hicolor/128x128/apps/neomutt.png`
`contrib/logo/neomutt-256.png` → `/usr/share/icons/hicolor/256x256/apps/neomutt.png`
`contrib/logo/neomutt.svg` → `/usr/share/icons/hicolor/scalable/apps/neomutt.svg`

**Comment by yuzibo** [comment]
Yeah. These icons should be put into `/usr/share/icons/hicolor/scalable/apps/` directory or its subdirectory.
That is the fix will be finished by neomutt Debian maintainer?

**Comment by dcpurton** [comment]
The Debian appstream page also warns about missing metadata. Perhaps creating one of these and automatically generating the `.desktop` file from that would be a better plan? Seems unwise to have similar information duplicated.

**Comment by dcpurton** [comment]
I opened a discussion topic to think about this at #3287.

**Comment by yuzibo** [comment]
Greate, thank you

**Comment by roccoblues** [comment]
-> https://github.com/neomutt/neomutt/discussions/3287
